__tests__/app.test.js

```diff
@@ -62,7 +62,8 @@ describe('GET /api/articles/:article_id', () => {
                 topic: expect.any(String),
                 created_at: expect.any(String),
                 votes: expect.any(Number),
-                article_img_url: expect.any(String)
+                article_img_url: expect.any(String),
+                comment_count: expect.any(Number)
```

Comment:
Ideally when adding new behaviour to existing endpoints, this should be done in a new test without the previous test breaking. If the previous test did break then the test was brittle. It is fine to have this here but ideally it should be in its own separate test as it is a new behaviour.